src/Logger.php

```diff
@@ -95,10 +95,14 @@ private function send(LogLevel $level, string|\Stringable $message, array $conte
                 $attrs = [];
 
                 foreach ($context as $key => $value) {
-                    $attrs[] = new LogAttrs(data: [
-                        'key' => $key,
-                        'value' => \is_string($value) ? $value : \json_encode($value),
-                    ]);
+                    try {
+                        $attrs[] = new LogAttrs(data: [
+                            'key' => $key,
+                            'value' => $this->prepareValue($value),
+                        ]);
+                    } catch (\JsonException) {
```

Comment:
An exception may be dropped from `Stringable::__toString()`. Should it be skipped in this case?